kotlin-spark-api/3.2/src/main/kotlin/org/jetbrains/kotlinx/spark/api/SparkHelper.kt

```diff
@@ -78,18 +83,39 @@ inline fun withSpark(builder: Builder, logLevel: SparkLogLevel = ERROR, func: KS
             KSparkSession(this).apply {
                 sparkContext.setLogLevel(logLevel)
                 func()
+                sc.stop()
```

Comment:
Is `sc` always started?

Reply:
Yes. Actually I might not have to manually stop it now that I think of it. spark.stop() stops the underlying sparkContext and JavaSparkContext is just a wrapper around the same sparkContext. So I'll remove this call.